src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -297,6 +297,7 @@ internal static async Task CopyToAsync(this Stream source, Stream destination, T
             }
 
             byte[] buffer = ArrayPool<byte>.Shared.Rent(StreamHelper.ChunkSize);
+            Memory<byte> bufMem = buffer.AsMemory(0, StreamHelper.ChunkSize);
```

Comment:
I don't understand why we need to reduce the buffer size. On fast network we can read 2, 3 and more chunks at time. Why do we limit the reading to one chunk at time if we already have large buffer in any case?